docs/organizations/using-a-coding-standard.md

```diff
@@ -0,0 +1,82 @@
+# Using a coding standard
+
+Create a coding standard on your organization to define and apply global tool and code pattern configurations consistently across your repositories. You can also apply the coding standard to new repositories automatically by defining the coding standard as default.
+
+Use the organization coding standard to ensure that Codacy analyzes all your repositories with the same tool and code pattern settings. For example, configure the organization coding standard to ensure that all your repositories and teams are following the same global security rules or coding conventions.
+
+If you then customize the tool and code patterns for a specific repository, Codacy warns you that the repository will stop following the coding standard and ask for your confirmation.
+
+## Creating a coding standard {: id="creating"}
+
+Organizations on Codacy can have one coding standard. To create a coding standard for your organization:
+
+1.  Open your organization **Coding standards** page, tab **Coding standards**.
+
+1.  Name your coding standard and click **Create standard**.
+
+    Optionally, select a repository that Codacy will use as a baseline to bootstrap the tool and pattern configurations for the new coding standard. This is useful if you already have a repository with code patterns configured and that you wish to use as a template for the coding standard.
+
+    ![Creating a new coding standard](images/coding-standard-create.png)
+
+1.  Select all the programming languages that the new coding standard will cover and click **Next: Tools and patterns**.
+
+    The coding standard will only include configurations for the tools that support at least one of the selected languages.
+
+    ![Selecting the languages for the coding standard](images/coding-standard-select-languages.png)
+
+1.  Configure the tools and patterns of the coding standard and click **Next: Select and apply to repositories**.
+
+    For each tool:
+
+    -   Toggle if Codacy will run the tool when analyzing your code
+    -   Configure the code patterns that the tool will use
```

Comment:
Consider mentioning what the "Recommended" column means. The marked patterns are the ones that Codacy enables by default, but this may not be clear to users, as observed by @fjrdomingues.

Reply:
@RaquelHipolito do you know why we decided to call it "Recommended" instead of "Default" or similar?

Reply:
I think the main reason it to avoid terms being used in other places. For example today we have the "Account default patterns" so it could create some confusion. We also have the feature to make a standard the default for the organization. With that, for "Codacy's default" we looked for another term, like our recommendation.

Reply:
I'm thinking of mentioning the "Recommended" code patterns only after [CY-4956](https://codacy.atlassian.net/browse/CY-4956) is released.

Reply:
I decided to mention the recommended tools and code patterns in this release after all, please check: https://github.com/codacy/docs/pull/823/files#diff-fafae958d2ea3ad6b16330ff9a2945458295b43f08eecff05dae54505b4161c3R38